recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -1,10 +1,9 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile, tools
+from conan.tools.scm import Version
 from conans.errors import ConanInvalidConfiguration
-import os
 
 required_conan_version = ">=1.33.0"
```

Comment:
The conan v2 API need a higher version (this is not the right one but just better)

```suggestion
required_conan_version = ">=1.43.0"
```